docker-compose.yml

```diff
@@ -131,17 +131,18 @@ services:
       lagoon.persistent: *lagoon-persistent-files
       lagoon.persistent.class: slow
 
-  mariadb:
+
+  database:
     build:
       context: .
-      dockerfile: .docker/mariadb.dockerfile
+      dockerfile: .docker/database.dockerfile
       args:
-        IMAGE: "${DREVOPS_DB_DOCKER_IMAGE:-uselagoon/mysql-8.0:25.1.0}" # Use custom database image (if defined) or fallback to standard database image.
+        IMAGE: "${DREVOPS_DB_DOCKER_IMAGE:-uselagoon/mysql-8.4:25.1.0}" # Use custom database image (if defined) or fallback to standard database image.
```

Comment:
To my earlier point of making local,dev and prod running compatible environment, I wold not go to 8.4